Woof/WoofTests/ViewModel/DetailPetSitterViewModel/DetailPetSitterViewModel+CopyToClipboardTextTests.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+
+// swiftlint:disable:next type_name
```

Comment:
Do you want to add a new rule to the `.swiftlint.yml`?
 Please remove comments, that don't contain documentation.

Reply:
Why do we need a new rule if SwiftLint provides opportunity to turn off rule for rare cases when it is needed?
The name of the test is a such case.

Reply:
Ok, but  production-ready code, as I know, shouldn't contain  regular comments.